apps/common/controllers/confirm.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const controllers = require('hof-confirm-controller');
+
+module.exports = class ConfirmController extends controllers {
+
+  locals(req, res) {
+    const content = req.rawTranslate('pages.confirm');
```

Comment:
yeah it sucks for 2 reasons, firstly because deepTranslate tries to lookup conditional values if a translation returns an object - this should be fixed in hof-middleware/deep-translate. Second reason as we discussed, we should look into markdown templates or similar. But also as you said, ¯\_(ツ)_/¯